iib/web/models.py

```diff
@@ -698,6 +704,15 @@ def _from_json(
                     ' the "overwrite_from_index_token" parameter is used'
                 )
 
+        distribution_scope = request_kwargs.pop('distribution_scope', None)
+        if distribution_scope:
```

Comment:
Shouldn't this be set to PROD by default?

Reply:
No IMO we want to keep the request what the user sent us right?

Reply:
Hmm maybe you're right, I'm not sure. @yashvardhannanavati what do you think?

Reply:
I agree with Shawn's point